.github/workflows/package-ci.yml

```diff
@@ -111,7 +111,7 @@ jobs:
         with:
           name: ${{ runner.os }}-${{ runner.arch }}-wheel
           path: dist/
-          if-no-file-found: error
+          if-no-files-found: error
```

Comment:
Can you extract that in a separate PR for fast merging ?

Reply:
Merged in #4214, would be remove when a rebase the branch